lambdas/process-cri-callback/src/test/java/uk/gov/di/ipv/core/processcricallback/pact/dcmawCri/ContractTest.java

```diff
@@ -0,0 +1,2788 @@
+                          "type": "FamilyName"
+                        }
+                      ]
+                    }
+                  ],
+                  "birthDate": [
+                    {
+                      "value": "1985-02-08"
+                    }
+                  ],
+                  "deviceId": [
+                    {
+                      "value": "fb03ce33-6cb4-4b27-b428-f614eba26dd0"
+                    }
+                  ],
+                  "address": [
+                    {
+                      "uprn": null,
+                      "organisationName": null,
+                      "subBuildingName": null,
+                      "buildingNumber": null,
+                      "buildingName": null,
+                      "dependentStreetName": null,
+                      "streetName": null,
+                      "doubleDependentAddressLocality": null,
+                      "dependentAddressLocality": null,
+                      "addressLocality": null,
+                      "postalCode": "CH1 1AQ",
+                      "addressCountry": null
+                    }
+                  ],
+                  "drivingPermit": [
+                    {
+                      "personalNumber": "DOE99802085J99FG",
+                      "expiryDate": "2023-01-18",
+                      "issueNumber": null,
+                      "issuedBy": "DVLA",
+                      "issueDate": "2022-05-29",
+                      "fullAddress": "WHATEVER STREET, WIRRAL, CH1 1AQ"
+                    }
+                  ]
+                },
+                "evidence": [
+                  {
+                    "type": "IdentityCheck",
+                    "txn": "ea2feefe-45a3-4a29-923f-604cd4017ec0",
+                    "strengthScore": 3,
+                    "validityScore": 2,
+                    "activityHistoryScore": 1,
+                    "checkDetails": [
+                      {
+                        "checkMethod": "vri",
+                        "identityCheckPolicy": "published",
+                        "activityFrom": "2022-05-29"
+                      },
+                      {
+                        "checkMethod": "bvr",
+                        "biometricVerificationProcessLevel": 3
+                      }
+                    ]
+                  }
+                ]
+              }
+            }
+            """;
+    // If we generate the signature in code it will be different each time, so we need to generate a
+    // valid signature (using https://jwt.io works well) and record it here so the PACT file doesn't
+    // change each time we run the tests.
+    private static final String VALID_DVLA_VC_SIGNATURE =
+            "MPwaYUsf5HBJ7Gp5oq5TJ71l1B2zfUXOrDeIXpotsnJpWsHonGV11yralObDqFM5UXCCrqemyCLQPWZ6z6seFg";
+
+    // 2099-01-01 00:00:00 is 4070908800 in epoch seconds
+    private static final String VALID_DVLA_VC_NO_GIVEN_NAME_BODY =
```

Comment:
This VC also appears to be missing a device id field compared to the other examples you've provided.

Not sure if this will be an issue given it's a failure case, but just wanted to check

Reply:
Good spot, I'll fix that up